cmake/Platform/Common/MSVC/Directory.Build.props

```diff
@@ -14,15 +14,18 @@ SPDX-License-Identifier: Apache-2.0 OR MIT
 @VCPKG_CONFIGURATION_MAPPING@
         <!-- Disable vcpkg to prevent include and linking paths from being added by the vcpkg integration -->
         <VcpkgEnabled>false</VcpkgEnabled>
+        <CodeAnalysisRuleSet>@LY_ROOT_FOLDER@\cmake\Platform\Common\MSVC\CodeAnalysis.ruleset</CodeAnalysisRuleSet>
```

Comment:
We can use `$(MSBuildThisFileDirectory)CodeAnalysis.ruleset` here to reference the CodeAnalysis.ruleset file in the same directory as this props file if we copy the CodeAnaylsis.ruleset file into the same location as the Directory.Build.props